<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QMainWindow" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: #0D0F3D;
 color:bisque;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="4">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:#22233D;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>protocole</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QWidget" name="widget_6" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_7" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="widget_11" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #22233D;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string> Target:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>INPUT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OUTPUT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FORWARD</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QWidget" name="widget_10" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #22233D;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>adresse MAC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2" colspan="2">
     <widget class="QWidget" name="widget_12" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #22233D;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_13" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string> adresse IP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_8" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #22233D;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string> Policy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>ACCEPT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DROP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>REJECT</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QWidget" name="widget_9" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #22233D;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>interface</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #0D0F3D;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">display: flex;
justify-content: center;
align-items:center;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>AJOUT DE REGLES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
